### ISSUE TITLE
WCAG 2.1: High contrast mode - 'Gear' icon's focus indicator has insufficient contrast

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-web#764]
## Describe the bug

Same as #760, in high contrast mode

## Screenshots

![image](https://user-images.githubusercontent.com/22068434/59233843-02876780-8b9f-11e9-80c7-d2737900997f.png)

![image](https://user-images.githubusercontent.com/22068434/59233960-8b060800-8b9f-11e9-9146-ab107d9b670d.png)

## Extension

- Accessibility Insights for Web - prod
- Browser Version: Chrome 75.0.3770.80
- Target Page: Any page

**Comment by ferBonnin** [comment]
@cheade thoughts for this?

**Comment by cheade** [comment]
@ferBonnin Yep! It's on my list to address today :)

**Comment by cheade** [comment]
Suggestion for high contrast mode based on my suggestion for #760:

![image](https://user-images.githubusercontent.com/40274682/59473561-cdc21d00-8df7-11e9-9162-d8b03b2eaaec.png)

It is #000000, with a dash gap of (2, 4). I've given some breathing room of 3px between the focus border and the icon itself.

**Comment by ferBonnin** [comment]
thanks Chelsea.
@iamrafan thoughts on this?

**Comment by iamrafan** [comment]
@ferBonnin I like this suggestion, as the focus indicator would be consistent with the drop down

![image](https://user-images.githubusercontent.com/22068434/59527404-57252e00-8e90-11e9-8a62-5e20063d7204.png)

**Comment by iamrafan** [comment]
Validated the fix in prod version 2.15.0
